snippets/oadp-ceph-cr-prerequisites.adoc

```diff
@@ -0,0 +1,11 @@
+// Text snippet included in the following modules:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: SNIPPET
+
+.Prerequisites
+
+* A stateful application is running in a separate namespace with persistent volume claims (PVCs) using CephFS as the provisioner.
+* The `StorageClass` and `VolumeSnapshotClass` CRs are defined for CephFS and OADP 1.2 Data Mover.
```

Comment:
Do we want to define CRs here, since it would likely be the first usage in the module?